moveit_py/src/moveit/moveit_core/planning_scene/planning_scene.cpp

```diff
@@ -70,6 +72,28 @@ moveit_msgs::msg::PlanningScene getPlanningSceneMsg(std::shared_ptr<planning_sce
   return planning_scene_msg;
 }
 
+bool saveGeometryToFile(std::shared_ptr<planning_scene::PlanningScene>& planning_scene,
+                        const std::string& file_path_and_name)
+{
+  std::ofstream save_file(file_path_and_name);
```

Comment:
Rename variable to `file`. It's not referring to an action.
```suggestion
  std::ofstream file(file_path_and_name);
```